sound/soc/sof/intel/hda-loader-skl.c

```diff
@@ -504,7 +506,7 @@ int hda_dsp_cl_boot_firmware_skl(struct snd_sof_dev *sdev)
 	hda_dsp_dump_skl(sdev, SOF_DBG_REGS | SOF_DBG_PCI | SOF_DBG_MBOX);
 
 	/* power down DSP */
-	hda_dsp_core_reset_power_down(sdev, HDA_DSP_CORE_MASK(0));
+	hda_dsp_core_reset_power_down(sdev, chip->cores_mask);
```

Comment:
Why would you use 2 cores by default? This makes no sense. It's not because you can that you should, it's horrible in terms of power consumption.

Reply:
iirc, this is copied exactly from SKL driver.

Reply:
@lgirdwood @zhuyingjiang I dont think the SKL driver does this. But even if it does, we have the means to bring up the cores when there are pipelines scheduled to run on them. I dont think it is necessary to power them all up at boot.

Reply:
OK, let's keep only one core enabled at first, so just remove the second commit is OK.